src/browser/modules/EditorInfo/utils.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * returns the string representation
```

Comment:
Given the neo4j property data type, transform the property value to string form.

Reply:
done